### ISSUE TITLE
Allow change import in es2015 mode

### ISSUE THREAD

**Issue by JoniJnm** [code-chris/lodash-loader#7]
Hi,

I need the change the replace of the import in es2015 mode (because of typescript config).

Can you add an option to change the replace by:
`return "import _" + lodashModule + " from \"lodash/" + lodashModule + "\";\n";`
?

Thanks!

**Comment by code-chris** [comment]
Hi,

so you need instead of this as result:
```
import * as _each from "lodash/each";
```

this here with `_each` as default export?
```
import _each from "lodash/each";
```

**Comment by JoniJnm** [comment]
Yes, thanks.

In typescript the replace is with `require('lodash/each')`

**Comment by code-chris** [comment]
Yes, I can implement this different behavior.

**Comment by code-chris** [comment]
@JoniJnm Released in 1.1.0

**Comment by JoniJnm** [comment]
Thanks!
